src/atoms/Button/Button.tsx

```diff
@@ -7,16 +7,18 @@ import Typography from '../Typography';
 
 interface Props {
   large?: boolean;
+  secondary?: boolean;
 }
 
 export const Button = styled(Typography)<Props>`
-  background: ${props => props.theme.primary};
-  border: none;
+  background: ${props => (props.secondary ? 'none' : props.theme.primary)};
+  border: ${props =>
+    props.secondary ? '1px solid' + props.theme.primary : 'none'};
   border-radius: ${borderRadius};
-  color: white;
+  color: ${props => (props.secondary ? props.theme.primary : 'white')};
   font-size: ${scale(0)};
   ${padding(scale(-1), scale(2))};
-  ${transitions(['opacity', 'background'], transitionDuration)};
+  ${transitions(['opacity', 'background, color'], transitionDuration)};
```

Comment:
I think you want `'background', 'color'` rather than `'background, color'`

Reply:
Oh good catch! That is correct.